src/nunavut/lang/cpp/templates/_fields.j2

```diff
@@ -11,4 +11,7 @@
     {% endif -%}
     {{ field.doc | block_comment('cpp-doxygen', 4, 120) }}
     {{ field.data_type | declaration }} {{ field | id }};
+    {%- if field.data_type is VariableLengthArrayType %}
```

Comment:
```suggestion
    {%- if field.data_type is ArrayType %}
```

On second thought, there is no point specializing this for variable-length array types only, I think.

Reply:
I think, there is value in consistency in providing this constant for fixed-sized arrays, yes.